Docs9233/disable option (#25714)
DataDog/documentation: content/en/metrics/summary.md

{{< img src="metrics/summary/facets4.png" alt="Metrics Facet Panel" style="width:75%;">}}

## Configuration of multiple metrics 

> Clicking on **Configure Metrics** gives you three options that you can use to configure multiple metrics at a time: 

Reviewer: ```suggestion
Clicking on **Configure Metrics** gives you multiple options that you can use to configure more than one metric at a time: 
```

Author: I'm not necessarily opposed to this update, but rather curious as to what the rationale behind it.

Reviewer: Basically, I try to avoid specific numbers whenever I can, because in future updates (say, adding something to the list of ways you can do it), changing the intro number might get missed.

Reviewer: Thanks for asking! I sometimes forget that I should explain my reasoning for stuff like this. :)

Author: Thanks for clarifying!

> {{< img src="metrics/summary/configurationbuttons10-11-2024.png" alt="Bulk Configuration Buttons" style="width:75%;">}}

Reviewer 2: This image is strangely much smaller than the GIF of manage tags right above it. Can we make it larger? as it's hard to see the text without clicking into the image?

Author: Thanks for that feedback, I added a consistent width for all `img` resources in this section in [342324b](https://github.com/DataDog/documentation/pull/25714/commits/342324b76936f11374653f083fc49106417cad56)

> {{< img src="metrics/summary/configurationbuttons10-11-2024.png" alt="Bulk Configuration Buttons" style="width:100%;">}}

Reviewer: ```suggestion
{{< img src="metrics/summary/configurationbuttons10-11-2024.png" alt="Bulk configuration buttons" style="width:100%;">}}
```

* **Manage tags**: Configure tags on multiple custom metrics matching a namespace using Metrics without Limits™.

{{< img src="metrics/summary/bulkconfig_new-compressed.mp4" alt="Bulk Metric Tag Configuration" video="true" style="width:100%;" >}}

> * **Enable percentiles**: Manage percentile aggregations for multiple distribution metrics.

Reviewer 2: ```suggestion
* **Enable/Disable percentiles**: Manage percentile aggregations across multiple distribution metrics.
```

> {{< img src="metrics/summary/percentile_aggregations_toggle.png" alt="Toggle to manage percentile aggregations" style="width:100%;">}}

Reviewer: Is this screenshot materially different from the one you used for percentile aggregations on the other page?

> * **Enable historical metrics**: Manage the ingestion of historical metric data. See the [Historical Metrics Ingestion page][30] for more information.

Reviewer 2: ```suggestion
* **Enable/Disable historical metrics ingestion**: Manage the ingestion of historical metric data. See the [Historical Metrics Ingestion page][30] for more information.
```

[27]: /universal_service_monitoring/
[28]: /logs/
[29]: /service_catalog/scorecards/
[30]: /metrics/custom_metrics/historical_metrics/#bulk-configuration-for-multiple-metrics

> [31]: /metrics/distributions/

Reviewer: I suggest adding the specific heading anchor here
